src/pages/LandingPage.js

```diff
@@ -67,7 +71,39 @@ const makeDocLinks = _.map(
 )
 
 const LandingPage = () => {
+  const { isSignedIn } = useStore(authStore)
+  const [billingProjects, setBillingProjects] = useState()
+  const signal = useCancellation()
+
+  useEffect(() => {
+    const loadProjects = withErrorIgnoring(async () => {
+      const projects = await Ajax(signal).Billing.listProjects()
+      setBillingProjects(projects)
+    })
+    if (isSignedIn) {
+      loadProjects()
+    }
+  }, [isSignedIn, setBillingProjects, signal])
+
   return h(HeroWrapper, { bigSubhead: true }, [
+    isTerra() && !_.isUndefined(billingProjects) && _.isEmpty(billingProjects) && div({
```

Comment:
@admull the decision to not show this for white-label sites is somewhat interesting. Users on those sites still have to deal with (Terra) billing projects before they can do anything, but I get that our current message is very "Terra-centric", so that is perhaps why we aren't showing it for other sites like eLwazi?

Reply:
I'm also interested in understanding the plan on how we'll handle white-label sites in this case.